src/server.js

```diff
@@ -12,7 +12,8 @@ function runServer() {
     const app = new Koa()
     app.use(serve('static'))
     app.use(async (context, next) => {
-        context.set('Access-Control-Allow-Origin', `http://localhost:${port.value}`)
+        context.set('Access-Control-Allow-Origin', `*`)
+        // context.set('Access-Control-Allow-Origin', `http://localhost:${port.value}`)
```

Comment:
Please note CORS can be more restrictive if running the bee-dashboard directly through bee-desktop